<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>750</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Connection</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>750</width>
      <height>600</height>
     </rect>
    </property>
    <property name="tabPosition">
     <enum>QTabWidget::North</enum>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Connecting</string>
     </attribute>
     <widget class="QLabel" name="l_login">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>130</y>
        <width>60</width>
        <height>25</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>18</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Login</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="edit_login">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>130</y>
        <width>191</width>
        <height>25</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>18</pointsize>
       </font>
      </property>
      <property name="text">
       <string extracomment="Ehter login">admin</string>
      </property>
     </widget>
     <widget class="QLabel" name="l_password">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>190</y>
        <width>91</width>
        <height>25</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>18</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Password</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="edit_password">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>190</y>
        <width>191</width>
        <height>25</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>18</pointsize>
       </font>
      </property>
      <property name="text">
       <string extracomment="Enter password">password</string>
      </property>
      <property name="echoMode">
       <enum>QLineEdit::Password</enum>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>260</y>
        <width>301</width>
        <height>81</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>18</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Connect</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Clients</string>
     </attribute>
     <widget class="QTableView" name="table">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>90</y>
        <width>661</width>
        <height>361</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>470</y>
        <width>41</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Page</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="sb_page_number">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>490</y>
        <width>181</width>
        <height>22</height>
       </rect>
      </property>
      <property name="buttonSymbols">
       <enum>QAbstractSpinBox::UpDownArrows</enum>
      </property>
      <property name="minimum">
       <number>1</number>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>470</y>
        <width>111</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Records for page</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="sb_page_size">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>490</y>
        <width>201</width>
        <height>22</height>
       </rect>
      </property>
      <property name="minimum">
       <number>1</number>
      </property>
      <property name="maximum">
       <number>15</number>
      </property>
      <property name="value">
       <number>11</number>
      </property>
     </widget>
     <widget class="QWidget" name="gridLayoutWidget">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>0</y>
        <width>531</width>
        <height>83</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="1" column="1">
        <widget class="QLineEdit" name="lastName_filter"/>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="label_4">
         <property name="font">
          <font>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>First Name </string>
         </property>
        </widget>
       </item>
       <item row="1" column="3">
        <widget class="QComboBox" name="sex_filter">
         <property name="currentText">
          <string>f</string>
         </property>
         <item>
          <property name="text">
           <string>f</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string/>
          </property>
         </item>
         <item>
          <property name="text">
           <string>m</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLineEdit" name="firstName_filter"/>
       </item>
       <item row="0" column="1">
        <widget class="QLabel" name="label_5">
         <property name="font">
          <font>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Last Name </string>
         </property>
        </widget>
       </item>
       <item row="0" column="3">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Sex </string>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QLabel" name="label_6">
         <property name="font">
          <font>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Birth Date</string>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QDateEdit" name="dateEdit_start">
           <property name="date">
            <date>
             <year>1950</year>
             <month>1</month>
             <day>1</day>
            </date>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QDateEdit" name="dateEdit_finish">
           <property name="date">
            <date>
             <year>2020</year>
             <month>1</month>
             <day>1</day>
            </date>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QPushButton" name="pushButton_2">
      <property name="geometry">
       <rect>
        <x>580</x>
        <y>1</y>
        <width>111</width>
        <height>81</height>
       </rect>
      </property>
      <property name="text">
       <string>Search</string>
      </property>
     </widget>
     <widget class="QPushButton" name="addclientbutton">
      <property name="geometry">
       <rect>
        <x>560</x>
        <y>470</y>
        <width>131</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>Add new Client</string>
      </property>
     </widget>
     <widget class="QPushButton" name="submit_button">
      <property name="geometry">
       <rect>
        <x>450</x>
        <y>470</y>
        <width>101</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>Edit </string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>Doctors</string>
     </attribute>
     <widget class="QTableView" name="doctors_table">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>90</y>
        <width>721</width>
        <height>371</height>
       </rect>
      </property>
     </widget>
     <widget class="QWidget" name="gridLayoutWidget_2">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>0</y>
        <width>551</width>
        <height>83</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <widget class="QLabel" name="label_16">
         <property name="text">
          <string>First Name</string>
         </property>
        </widget>
       </item>
       <item row="0" column="3">
        <widget class="QLabel" name="label_15">
         <property name="text">
          <string>Sex</string>
         </property>
        </widget>
       </item>
       <item row="1" column="3">
        <widget class="QComboBox" name="doctor_sex_edit">
         <property name="currentText">
          <string>f</string>
         </property>
         <item>
          <property name="text">
           <string>f</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>m</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string/>
          </property>
         </item>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLineEdit" name="doctor_firstname_edit"/>
       </item>
       <item row="0" column="2">
        <widget class="QLabel" name="label_18">
         <property name="text">
          <string>Birth Date</string>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <widget class="QDateEdit" name="doctor_date_start_edit">
           <property name="date">
            <date>
             <year>1950</year>
             <month>1</month>
             <day>1</day>
            </date>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QDateEdit" name="doctor_finish_date_edit">
           <property name="date">
            <date>
             <year>2020</year>
             <month>1</month>
             <day>1</day>
            </date>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="doctor_last_name_edit"/>
       </item>
       <item row="0" column="1">
        <widget class="QLabel" name="label_17">
         <property name="text">
          <string>Last Name</string>
         </property>
        </widget>
       </item>
       <item row="0" column="4">
        <widget class="QLabel" name="label_19">
         <property name="text">
          <string>Specialisation</string>
         </property>
        </widget>
       </item>
       <item row="1" column="4">
        <widget class="QLineEdit" name="spec_filter"/>
       </item>
      </layout>
     </widget>
     <widget class="QLabel" name="label_13">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>470</y>
        <width>60</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Page</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="page_number">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>490</y>
        <width>161</width>
        <height>22</height>
       </rect>
      </property>
      <property name="minimum">
       <number>1</number>
      </property>
     </widget>
     <widget class="QLabel" name="label_14">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>470</y>
        <width>111</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Records for page</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="page_size">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>490</y>
        <width>161</width>
        <height>22</height>
       </rect>
      </property>
      <property name="minimum">
       <number>1</number>
      </property>
      <property name="maximum">
       <number>15</number>
      </property>
      <property name="value">
       <number>11</number>
      </property>
     </widget>
     <widget class="QPushButton" name="edit_doctor_button">
      <property name="geometry">
       <rect>
        <x>430</x>
        <y>470</y>
        <width>121</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>Edit</string>
      </property>
     </widget>
     <widget class="QPushButton" name="add_new_doctor_button">
      <property name="geometry">
       <rect>
        <x>560</x>
        <y>470</y>
        <width>141</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>Add new Doctor</string>
      </property>
     </widget>
     <widget class="QPushButton" name="doctor_filter_button">
      <property name="geometry">
       <rect>
        <x>600</x>
        <y>0</y>
        <width>121</width>
        <height>81</height>
       </rect>
      </property>
      <property name="text">
       <string>Search</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_4">
     <attribute name="title">
      <string>Salons</string>
     </attribute>
     <widget class="QTableView" name="salon_table">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>100</y>
        <width>711</width>
        <height>371</height>
       </rect>
      </property>
     </widget>
     <widget class="QWidget" name="gridLayoutWidget_5">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>10</y>
        <width>511</width>
        <height>80</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout_5">
       <item row="0" column="3">
        <widget class="QLabel" name="label_23">
         <property name="text">
          <string>StreetName</string>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QLabel" name="label_22">
         <property name="text">
          <string>City</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLabel" name="label_21">
         <property name="text">
          <string>Country</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="label_20">
         <property name="text">
          <string>Salon Name</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLineEdit" name="salon_name_edit"/>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="country_edit"/>
       </item>
       <item row="1" column="2">
        <widget class="QLineEdit" name="city_edit"/>
       </item>
       <item row="1" column="3">
        <widget class="QLineEdit" name="street_edit"/>
       </item>
      </layout>
     </widget>
     <widget class="QPushButton" name="activate_salon_filter_button">
      <property name="geometry">
       <rect>
        <x>572</x>
        <y>10</y>
        <width>131</width>
        <height>81</height>
       </rect>
      </property>
      <property name="text">
       <string>Search</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_24">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>480</y>
        <width>60</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Page</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_25">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>480</y>
        <width>111</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Records for page</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="salon_page_number">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>500</y>
        <width>181</width>
        <height>22</height>
       </rect>
      </property>
      <property name="minimum">
       <number>1</number>
      </property>
     </widget>
     <widget class="QSpinBox" name="salon_page_size">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>500</y>
        <width>191</width>
        <height>22</height>
       </rect>
      </property>
      <property name="minimum">
       <number>1</number>
      </property>
      <property name="maximum">
       <number>15</number>
      </property>
      <property name="value">
       <number>11</number>
      </property>
     </widget>
     <widget class="QPushButton" name="edit_salon_button">
      <property name="geometry">
       <rect>
        <x>440</x>
        <y>475</y>
        <width>121</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>Edit</string>
      </property>
     </widget>
     <widget class="QPushButton" name="insert_salon_button">
      <property name="geometry">
       <rect>
        <x>570</x>
        <y>475</y>
        <width>141</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>Add new Salon</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_5">
     <attribute name="title">
      <string>Appointments</string>
     </attribute>
     <widget class="QTableView" name="app_table">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>100</y>
        <width>711</width>
        <height>361</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_26">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>470</y>
        <width>60</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Page</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_27">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>470</y>
        <width>111</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Records for page</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="app_page_number">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>490</y>
        <width>151</width>
        <height>22</height>
       </rect>
      </property>
      <property name="minimum">
       <number>1</number>
      </property>
     </widget>
     <widget class="QSpinBox" name="app_page_size">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>490</y>
        <width>151</width>
        <height>22</height>
       </rect>
      </property>
      <property name="minimum">
       <number>1</number>
      </property>
      <property name="maximum">
       <number>15</number>
      </property>
      <property name="value">
       <number>11</number>
      </property>
     </widget>
     <widget class="QPushButton" name="app_info_button">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>470</y>
        <width>171</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>Additional Info</string>
      </property>
     </widget>
     <widget class="QPushButton" name="app_add_button">
      <property name="geometry">
       <rect>
        <x>519</x>
        <y>470</y>
        <width>201</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>Add new Appointment</string>
      </property>
     </widget>
     <widget class="QWidget" name="gridLayoutWidget_6">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>10</y>
        <width>561</width>
        <height>80</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout_6">
       <item row="0" column="1">
        <widget class="QLabel" name="label_29">
         <property name="text">
          <string>Doctor ID</string>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QLabel" name="label_30">
         <property name="text">
          <string>Salon Name</string>
         </property>
        </widget>
       </item>
       <item row="0" column="3">
        <widget class="QLabel" name="label_31">
         <property name="text">
          <string>Date</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="label_28">
         <property name="text">
          <string>Client ID</string>
         </property>
        </widget>
       </item>
       <item row="0" column="4">
        <widget class="QLabel" name="label_32">
         <property name="text">
          <string>Status</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLineEdit" name="app_client_id_edit"/>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="app_doctor_id_edit"/>
       </item>
       <item row="1" column="2">
        <widget class="QLineEdit" name="app_salon_name_edit"/>
       </item>
       <item row="1" column="4">
        <widget class="QLineEdit" name="app_status_edit"/>
       </item>
       <item row="1" column="3">
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <item>
          <widget class="QDateEdit" name="app_first_date_edit">
           <property name="date">
            <date>
             <year>1950</year>
             <month>1</month>
             <day>1</day>
            </date>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QDateEdit" name="app_lat_date_edit">
           <property name="date">
            <date>
             <year>2030</year>
             <month>1</month>
             <day>1</day>
            </date>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QPushButton" name="app_filter_button">
      <property name="geometry">
       <rect>
        <x>592</x>
        <y>11</y>
        <width>131</width>
        <height>81</height>
       </rect>
      </property>
      <property name="text">
       <string>Search</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>750</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
